modules/using-a-hugging-face-prompt-injection-detector-with-the-guardrails-orchestrator.adoc

```diff
@@ -2,7 +2,7 @@
 
 ifdef::context[:parent-context: {context}]
 [id="using-a-hugging-face-prompt-injection-detector-with-guardrails-orchestrator_{context}"]
-= Using a Hugging Face Prompt Injection detector with the Guardrails Orchestrator
+= Preventing Prompt Injection by using a Hugging Face Prompt Injection detector
```

Comment:
If this title is OK, I'll go ahead and update the id=, but that creates a kind of pain of needing to update it in multiple places, so I want to be sure the titles are okay from stakeholder POV before I make deep changes.

Reply:
"Mitigating" is usually a safer word. "Preventing" is stronger and may not be 100% guranteed.